test/CoveragePool.test.js

```diff
@@ -477,20 +477,19 @@ describe("CoveragePool", () => {
 
     context("when underwriter did not delegate voting", () => {
       beforeEach(async () => {
-        // Underwriter deposits into the asset pool
-        await collateralToken.mint(underwriter1.address, to1e18(400))
+        await collateralToken.mint(thirdParty.address, to1e18(400))
         await collateralToken
-          .connect(underwriter1)
+          .connect(thirdParty)
           .approve(assetPool.address, to1e18(400))
-        await assetPool.connect(underwriter1).deposit(to1e18(400))
+        await assetPool.connect(thirdParty).deposit(to1e18(400))
 
         lastFinalizedBlock = (await lastBlockNumber()) - 1
       })
 
       it("should return zero", async () => {
         expect(
           await collateralToken.getPastVotes(
-            underwriter1.address,
+            thirdParty.address,
             lastFinalizedBlock
           )
         ).to.equal(0)
```

Comment:
@pdyraga What's the purpose of this test? `underwriter1` (now `thirdParty`) was minting 400 tokens, approving the asset pool and depositing all of them there; after this, we check that they have 0 votes, but this is going to happen always, regardless of them having delegated voting or not, because they have 0 tokens in the end anyway.

Reply:
>  but this is going to happen always, regardless of them having delegated voting or not, because they have 0 tokens in the end anyway.

I am not following this part. This is how I understand this test:

Deployer minted 400 tokens to `thirdParty`. `thirdParty` deposited 400 tokens to the coverage pool and received COV tokens. We check that the `thirdParty` has 0 voting power because they have not delegated this voting power to anyone - including self - even though they have 400 COV.

In other words, this test ensures that depositing to the coverage pool is not enough to receive voting power. One must delegate voting power on the COV they have.

Reply:
tl;dr; @cygnusv was right 🙈 https://github.com/keep-network/coverage-pools/pull/206/commits/31d3af2039adc8a556e2ed125b28af4a7a16a371